pypi_formula_mappings.json

```diff
@@ -266,6 +266,9 @@
   "flit": {
     "exclude_packages": ["certifi", "docutils", "flit-core"]
   },
+  "fonttools": {
+    "extra_packages": ["brotli", "setuptools"]
```

Comment:
Only `setuptools` is necessary here. But actually it is a build dependency only, so I am wondering if we could not get rid of it, see for example: https://github.com/Homebrew/homebrew-core/pull/163143#discussion_r1493770952

Reply:
brotli is an optional package, so unless I have it here, it's removed when I try to update the resources.